pluma/pluma-prefs-manager-app.c

```diff
@@ -713,6 +729,27 @@ pluma_prefs_manager_app_init (void)
 				"changed",
 				G_CALLBACK (pluma_prefs_manager_lockdown_changed),
 				NULL);
+
+		g_signal_connect (pluma_prefs_manager->settings,
+				"changed::" GPM_SPACE_DRAWER_SPACE,
+				G_CALLBACK (pluma_prefs_manager_draw_spaces_changed),
+				NULL);
+
+		g_signal_connect (pluma_prefs_manager->settings,
+				"changed::" GPM_SPACE_DRAWER_TAB,
+				G_CALLBACK (pluma_prefs_manager_draw_tabs_changed),
+				NULL);
+
+		g_signal_connect (pluma_prefs_manager->settings,
+				"changed::" GPM_SPACE_DRAWER_NEWLINE,
+				G_CALLBACK (pluma_prefs_manager_draw_newlines_changed),
+				NULL);
+
+		g_signal_connect (pluma_prefs_manager->settings,
+				"changed::" GPM_SPACE_DRAWER_NBSP,
+				G_CALLBACK (pluma_prefs_manager_draw_nbsp_changed),
+				NULL);
```

Comment:
Same here, the lines below the opening `(` need two more spaces...

Reply:
Same here: few lines above i see:
```
		g_signal_connect (pluma_prefs_manager->lockdown_settings,
				"changed",
				G_CALLBACK (pluma_prefs_manager_lockdown_changed),
				NULL);
```